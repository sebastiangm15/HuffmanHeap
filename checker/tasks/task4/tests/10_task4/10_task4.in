12
11 M9
12 M10
2 M1
3 M2
8 M5
6 M6
4 M7
5 M3
7 M4
9 M8
10 M11
13 M12
5
M5
M10
M2
M7
M12
