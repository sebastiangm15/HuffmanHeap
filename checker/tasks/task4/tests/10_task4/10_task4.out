M7M1M2M8M11M3M6M9M10M12M4M5
